{"name":"loginFail","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [fav] but found [fav_user]","trace":"java.lang.AssertionError: expected [fav] but found [fav_user]\n\tat org.testng.Assert.fail(Assert.java:110)\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\n\tat org.testng.Assert.assertEquals(Assert.java:131)\n\tat org.testng.Assert.assertEquals(Assert.java:655)\n\tat org.testng.Assert.assertEquals(Assert.java:665)\n\tat com.browserstack.test.suites.login.LoginFailTest.loginFail(LoginFailTest.java:20)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:567)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:658)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:219)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:923)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\n\tat org.testng.TestRunner.run(TestRunner.java:603)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\n\tat org.testng.SuiteRunner.access$000(SuiteRunner.java:32)\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:467)\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:831)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"platform","value":"1"},{"name":"BSPlatform","value":"iPhone 13, 15, Chromium"},{"name":"BSPlatform","value":"iPhone 13, 15, Chromium"},{"name":"BSPlatform","value":"iPhone 13, 15, Chromium"}],"start":1716463856677,"stop":1716463858809,"uuid":"2b2fe421-d6c9-41e5-bd31-c5b847337a8c","historyId":"66a4984d31c1a164a589fe326ab96175setUp@1","fullName":"com.browserstack.test.suites.login.LoginFailTest.loginFail","labels":[{"name":"package","value":"com.browserstack.test.suites.login.LoginFailTest"},{"name":"testClass","value":"com.browserstack.test.suites.login.LoginFailTest"},{"name":"testMethod","value":"loginFail"},{"name":"parentSuite","value":"Single"},{"name":"suite","value":"BStackDemo 3-1"},{"name":"subSuite","value":"com.browserstack.test.suites.login.LoginFailTest"},{"name":"host","value":"SiddhiRao-FVFGWQ7VQ05D"},{"name":"thread","value":"4747@SiddhiRao-FVFGWQ7VQ05D.setUp@1(56)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}